microsoft-edge/webview2/how-to/prerelease-testing.md

```diff
@@ -0,0 +1,189 @@
+
+
+<!-- ====================================================================== -->
+## Set the preview channel by using ChannelSearchKind
+
+The channel search kind controls the channel search order.  By default, the WebView2 loader searches for binaries from most-stable to least-stable (Canary), using the first binary that's found:
+
+```
+WebView2 Runtime (Stable) ->  Edge Beta -> Edge Dev -> Edge Canary
+```
+
+To do prerelease testing, reverse the channel search order, so that the loader looks for binaries in order from least-stable (Canary) to most-stable:
+
+```
+Edge Canary -> Edge Dev -> Edge Beta -> WebView2 Runtime (Stable)
+```
+
+The channel search kind can be set per-app through an API, a registry key, an environment variable, or a group policy.  Using the API is the recommended approach.
+
+The value `1` indicates reversed search order; that is, from least-stable (Edge Canary) to most-stable.
+
+
+##### [API](#tab/api)
+
+Using the API is the recommended approach.
+
+By default, the `CoreWebView2EnvironmentOptions.ChannelSearchKind` property is `CoreWebView2ChannelSearchKind.MostStable` (an enum value).  Instead, reverse the search order by setting the `CoreWebView2EnvironmentOptions.ChannelSearchKind` property to `CoreWebView2ChannelSearchKind.LeastStable`.
+
+.NET:
+* [CoreWebView2EnvironmentOptions.ChannelSearchKind Property](/dotnet/api/microsoft.web.webview2.core.corewebview2environmentoptions.channelsearchkind)
+* [CoreWebView2ChannelSearchKind Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2channelsearchkind)
+
+WinRT:
+* [CoreWebView2EnvironmentOptions.ChannelSearchKind Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environmentoptions#channelsearchkind)
+* [CoreWebView2ChannelSearchKind Enum](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2channelsearchkind)
+
+Win32:
+* [ICoreWebView2ExperimentalEnvironmentOptions::get_ChannelSearchKind](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalenvironmentoptions#get_channelsearchkind)
+* [ICoreWebView2ExperimentalEnvironmentOptions::put_ChannelSearchKind](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalenvironmentoptions#put_channelsearchkind)
+* [COREWEBVIEW2_CHANNEL_SEARCH_KIND enum](/microsoft-edge/webview2/reference/win32/webview2experimental-idl#corewebview2_channel_search_kind)
+
+
+##### [Registry key](#tab/registry-key)
+
+```reg
+REG ADD <HKLM/HKCU>\Software\Policies\Microsoft\Edge\WebView2\ChannelSearchKind /v WebView2APISample.exe /t REG_DWORD /d 1
+```
+
+Replace `WebView2APISample.exe` with your own app executable name or the application user model ID.
+
+
+##### [Environment variable](#tab/environment-variable)
+
+Name: `WEBVIEW2_CHANNEL_SEARCH_KIND`  
+Value: `1`
+
+Note that the environment variable will be applied to all apps that use WebView2 on the machine, unlike the registry key, which can be set per app.
```

Comment:
Would it be more accurate to say:

Note that you are not able to specify the target app when using the environment variable. Therefore, when set as a global environment, it will affect all apps that use WebView2 on the machine.

Reply:
Rendered articles for review: 
* **Test upcoming APIs and features** (update to use new API)
   * https://review.learn.microsoft.com/microsoft-edge/webview2/how-to/set-preview-channel?branch=pr-en-us-3053
* **Prerelease testing** (new article)
   * https://review.learn.microsoft.com/microsoft-edge/webview2/how-to/prerelease-testing?branch=pr-en-us-3053
* **Self-hosting** (new article)
   * https://review.learn.microsoft.com/microsoft-edge/webview2/how-to/self-hosting?branch=pr-en-us-3053

For "Outdated" comments, to see latest Markdown/commits, use fresh Files Changed tab:
https://github.com/MicrosoftDocs/edge-developer/pull/3053/files

Reply:
closing comment b/c new location of this info is:
https://review.learn.microsoft.com/microsoft-edge/webview2/how-to/set-preview-channel?branch=pr-en-us-3053&tabs=environment-variable - 
"The environment variable will be applied to all apps that use WebView2 on the machine, unlike the registry key, which can be set per app."

The objected-to wording in Outdated markdown was: 
Note that 
the environment variable will be applied to all apps that use WebView2 on the machine, unlike the registry key, which can be set per app.

Thus the orig wording remains, against the suggestion from Nic, which is:
"You are not able to specify the target app when using the environment variable. Therefore, when set as a global environment, it will affect all apps that use WebView2 on the machine."

I plan to create a fresh Comment to suggest that change, in set-preview-channel.md.  
@champnic @victorhuangwq

Reply:
Fresh [comment](https://github.com/MicrosoftDocs/edge-developer/pull/3053/files#r1571503671) that replaces the present, Outdated comment